backend/scraper/scrape_walmart.py

```diff
@@ -19,20 +20,31 @@ def __init__(self):
             chrome_options = Options()
             chrome_options.add_argument('--headless')
             try:
-                driver = r'./chromedriver.exe'
-                self.driver = webdriver.Chrome(driver, options=chrome_options)
+                self.driver = webdriver.Chrome(ChromeDriverManager().install())
+                self.driver.get("https://www.walmart.com/")
+                #self.driver.get = webdriver.Chrome(driver, options=chrome_options)
+                # Get cookies for request session - needed for getting each product
+                cookies = self.driver.get_cookies()
+                self.driver.quit()
+                 # Creating a session
+                self.session = requests.Session()
+                # PLace cookies in the session cookie jar
+                [self.session.cookies.set(cookie['name'], cookie['value']) for cookie in cookies]
+                
+                
             except:
                 sys.exit('Chrome web driver is missing')
-        elif browser.lower() == 'firefox':
-            try:
-                self.driver = webdriver.Firefox(executable_path=r'.\geckodriver.exe')
-            except:
-                sys.exit("Web Driver is missing")
+        # elif browser.lower() == 'firefox':

```

Comment:
Also, since the Chrome driver is the only one that works, would you mind removing the command-line option and just using chrome by default?